### PR TITLE
Add ITS/MFT time-dependent DeadMaps calibration

### PR DESCRIPTION
**Pull request by shahor02**

### CODE REVIEW

File: prodtests/full-system-test/aggregator-workflow.sh [hunk 1]
```diff
@@ -195,7 +197,14 @@ if [[ $AGGREGATOR_TASKS == BARREL_TF ]] || [[ $AGGREGATOR_TASKS == ALL ]]; then
...
     add_W o2-calibration-mean-vertex-calibration-workflow "" "MeanVertexCalib.tfPerSlot=$TFPERSLOTS_MEANVTX;MeanVertexCalib.maxTFdelay=$DELAYINTFS_MEANVTX"
   fi
-
+  # ITS
+  if [[ $CALIB_ITS_DEADMAP_TIME == 1 ]]; then
+     add_W o2-itsmft-deadmap-builder-workflow
+  fi
+  # MFT
+  if [[ $CALIB_MFT_DEADMAP_TIME == 1 ]]; then
+     add_W o2-itsmft-deadmap-builder-workflow  "--runmft"
```
Comment (shahor02) [inline prodtests/full-system-test/aggregator-workflow.sh]: @nicolovalle could you please check if some default options should be changed (e.g. `--skip-static-map`) ?

Comment (nicolovalle) [reply prodtests/full-system-test/aggregator-workflow.sh]: Dear @shahor02 , thank you for this.

Indeed, we can add `—skip-static-map` for MFT, *not for ITS*.

The reason is that he static part of the map is by construction included in the time-evolving one for MFT but not for ITS.

Comment (nicolovalle) [reply prodtests/full-system-test/aggregator-workflow.sh]: For all the other options the default are good as they are.

**Comment by shahor02** [comment]
@nicolovalle ok, added `--skip-static-map` for MFT only, @MauriceCoke please confirm that this ok.

**Comment by mcoquet642** [comment]
Hello @shahor02, I confirm that this option is ok for MFT

**Pull request merged**